c8run/main.go

```diff
@@ -71,24 +68,21 @@ func queryCamundaHealth(name string, url string) {
 		fmt.Println("Error: " + name + " did not start!")
 		os.Exit(1)
```

Comment:
Not in this PR. However, I would return the error:

"fmt.Errorf("Application %s did not start!", name)"

The caller can then handle what they want to do with that. Otherwise it is surprising that the function does not return an error in this case but exits.

You also already handle the case the same (your main.go exits on error and prints it)

Reply:
that's a good point. I returned an error in 967cbfb2200 instead of printing+exiting

Reply:
I'm not concerned about the weird function name. I'd wait for someone to complain about it before changing that.